client/grpc/tx.go

```diff
@@ -0,0 +1,126 @@
+package client
+
+import (
+	"context"
+	"github.com/bnb-chain/gnfd-go-sdk/types"
+	clitx "github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	xauthsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
+	"google.golang.org/grpc"
+)
+
+type TransactionClient interface {
+	BroadcastTx(msgs []sdk.Msg, txOpt *types.TxOption, opts ...grpc.CallOption) (*types.TxBroadcastResponse, error)
+	SendToken(req types.SendTokenRequest, txOpt *types.TxOption, opts ...grpc.CallOption) (*types.TxBroadcastResponse, error)
+}
+
+func (c *GreenfieldClient) BroadcastTx(msgs []sdk.Msg, txOpt *types.TxOption, opts ...grpc.CallOption) (*types.TxBroadcastResponse, error) {
+
+	txConfig := authtx.NewTxConfig(types.Cdc(), authtx.DefaultSignModes)
```

Comment:
only `SignMode_SIGN_MODE_EIP_712` is supported.

Reply:
changed